### PR TITLE
feat: add other patterns of default value

### LINKED ISSUE #23: Add other patterns of default value of argument
Now, in parser using only one pattern of default value of argument:
 `/\[default:` (.*?)\]/

So, default value will be parsed only if it looks like `[default: 70]`.

Please, add other patterns.

### PR DESCRIPTION
**Pull request by ifedchankau**
Add other patterns of default value

Closes #23

### CODE REVIEW

File: src/template/context.js [hunk 1]
```diff
@@ -16,7 +16,11 @@ const context = {
         },
     },
     regexp: {
-        defaultValue: '\\[default: (.*?)\\]',
+        defaultValue: [
+            '\\[default: (.*?)\\]',
+            '\\(default: (.*?)\\)',
+            '[\\s]+default:[\\s]+(.*?)([\\s]+|$)',
```
Comment (romanrostislavovich) [inline src/template/context.js]: I think, we can to combine them. Then It will not be necessary to change `setDescription ` function

**Review by romanrostislavovich — COMMENTED** [review]
Please, combine regex for defaultValue

File: src/template/context.js (hunk 1 above)
Comment (ifedchankau) [reply src/template/context.js]: @svoboda-rabstvo done

**Review by romanrostislavovich — APPROVED** [review]
LGTM

**Pull request merged**